src/platform/mhv/secondary-nav/containers/MhvSecondaryNav.jsx

```diff
@@ -27,19 +39,27 @@ export const mhvSecNavItems = [
     iconClass: 'fas fa-prescription-bottle',
     href: `/my-health/medications`,
   },
-  {
-    title: 'Records',
-    iconClass: 'fas fa-file-medical',
-    href: `/my-health/medical-records`,
-  },
+  medicalRecordsLink,
 ];
 
 /**
  * MHV secondary navigation bar.
  * @returns the navigation bar
  */
 const MhvSecondaryNav = () => {
-  return <MhvSecondaryNavMenu items={mhvSecNavItems} />;
+  const items = [...mhvSecNavItems];
+  const { loading, mhvTransitionalMedicalRecordsLandingPage } = useSelector(
+    state => state.featureToggles,
+  );
+
+  if (loading) return <></>;
+
+  if (mhvTransitionalMedicalRecordsLandingPage) {
```

Comment:
This code is dependent on the records item being last and we have already changed the order a couple of times. Could there be a check or safer way of doing this?

Reply:
The tests will catch it.

Reply:
Would you prefer this change?

```
  if (mhvTransitionalMedicalRecordsLandingPage) {
    items.push(transitionalMedicalRecordsLink);
  } else {
    items.push(medicalRecordsLink);
  }
```